src/main/java/seedu/address/model/person/Person.java

```diff
@@ -114,4 +114,10 @@ public String toString() {
                 .toString();
     }
 
+    public Person addTag(Tag tag) {
```

Comment:
Instead of modifying person itself, it might make sense to follow the convention made by the other commands. Look at https://github.com/AY2324S2-CS2103T-W13-2/tp/blob/ff8243bdf16a4e9aac2ec018720a4576a3118d3a/src/main/java/seedu/address/logic/commands/EditCommand.java#L91-L105 for the current convention

Reply:
I realise now that this is cumbersome and kinda counterintuitive. We should determine how to do this better at the level of the  person though, instead of just doing it for tags only.